k-distribution/tests/regression-new/map-symbolic-tests-haskell/assignment-26-spec.k.out

```diff
@@ -0,0 +1,5 @@
+<k>
+  assignmentResult ( 0 |-> 5
+  1 |-> 4
+  X:MyId |-> 3 ) ~> .
+</k>
```

Comment:
I think this gets simplified correctly.

On the Java backend, though, the expected result is:
```
<k>
  assignmentResult ( 1 |-> 4
  ( ( X |-> 3 ) [ 0 <- 5 ] ) )
</k>
```
It's also in the `failing` directory. So I don't know what to make of this test.